packages/govuk-frontend/src/govuk/govuk-frontend-component.mjs

```diff
@@ -39,8 +60,9 @@ export class GOVUKFrontendComponent {
    *
    * @internal
    * @param {Element | null} [$root] - HTML element to use for component
+   * @param {...ConfigType} configObjects - Configuration objects for component
    */
-  constructor($root) {
+  constructor($root, ...configObjects) {
```

Comment:
**issue** This looks troublesome in the long term if the component needs a 3rd or 4th argument. If in the long run we realise we need to accept more than one config object, that second parameter could accept an `Array` of configs, which we'd easily be able to distinguish from a single configuration object.

What prompted the use of rest parameters to receive a list of config objects?